sfy-buoy/sfy-artemis/src/main.rs

```diff
@@ -107,7 +107,7 @@ fn main() -> ! {
     info!("Setting up IOM and RTC.");
     delay.delay_ms(1_000u32);
 
-    let i2c4 = i2c::I2c::new(dp.IOM4, pins.d10, pins.d9, i2c::Freq::F400kHz);
+    let i2c4 = i2c::I2c::new(dp.IOM4, pins.d10, pins.d9, i2c::Freq::F100kHz);
```

Comment:
This decreases the communication frequency with the SD-card, should probably not be done?

Reply:
I am not sure. i2c4 is the notecarrier, and it was at 100kHz originally, so i figured i would change it back since it did not make a difference